app/models/complaint.rb

```diff
@@ -9,12 +9,26 @@ class Complaint
     initialContactDate: "Initial contact from complaint"
   }.with_indifferent_access.freeze
 
+  FORMATTED_STATUS = {
+    0 => "In Progress",
+    1 => "Closed",
+    2 => "Rec. Closure",
+    3 => "Rec. Reopening",
+    4 => "Reopened"
+  }.with_indifferent_access.freeze
```

Comment:
[question] in retrospect, maybe indifferent access doesn't matter here because I don't think numbers can be symbols...?